core/jobs/batch_jobs/suggestion_stats_computation_jobs.py

```diff
@@ -178,17 +408,26 @@ def _generate_stats(
                 content_word_count: int. The word count of the content of
                     the suggestion.
                 last_updated_date: str. When was the suggestion last updated.
+                created_date: str. When was the suggestion created.
         """
         # When opportunity is not available we leave the topic ID empty.
         topic_id = ''
         if opportunity is not None:
             topic_id = opportunity.topic_id
 
         for suggestion in suggestions:
-            key = (
-                suggestion_models
-                .TranslationContributionStatsModel.construct_id(
-                    suggestion.language_code, suggestion.author_id, topic_id))
+            if model == suggestion_models.TranslationContributionStatsModel:
+                key = model.construct_id(
+                    suggestion.language_code,
+                    suggestion.author_id,
+                    topic_id
+                )
+            else:
+                key = model.construct_id(
+                    suggestion.language_code,
+                    suggestion.final_reviewer_id,
+                    topic_id
+                )
```

Comment:
The language_code and topic_id remain the same, so let's create a variable for the user_id instead. e.g.

```
user_id = suggestion.author_id if model == suggestion_models.TranslationContributionStatsModel else suggestion.final_reviewer_id
key = model.construct_id(
                    suggestion.language_code,
                    user_id,
                    topic_id
                )
```

Ditto below.